Core/DISET/private/Service.py

```diff
@@ -577,4 +632,16 @@ def __endReportToMonitoring( self, initialWallTime, initialCPUTime ):
     cpuTime = stats[0] + stats[2] - initialCPUTime
     percentage = cpuTime / wallTime * 100.
     if percentage > 0:
-      self._monitor.addMark( 'CPU', percentage )
+      if self.activityMonitoring:
+        # ElasticDB
+        self.log["CPU"] = percentage
+        gLogger.info("Added CPU")
+        self.log["timestamp"] = time.time()
+
+        if "RunningThreads" in self.log:
```

Comment:
I do not understand why do you only add the record if `RunningThreads` is in it ?

Reply:
I observed that there were many logs of `Configuration Server` which were being added without `ThreadPoolContents` so that's why I did this.

Reply:
Sorry I do not understand what you mean. 
If there are logs (and they are not *logs* ) without the `ThreadPoolContents`, there must be a reason. But probably not a reason to ignore these records. And of course, if you check how this `ThreadPoolContents` is filled, you'll see that it's a period task every 30 seconds